theano/tensor/nnet/sigm.py

```diff
@@ -602,6 +602,7 @@ def local_exp_over_1_plus_exp(node):
             else:
                 # case: 1/(1+exp(x))
                 sigmoids.append(sigmoid(-t))
+            copy_stack_trace(node.outputs[0], sigmoids[-1])
```

Comment:
The issue seems to be that an optimization during the canonicalization phase, introducing the "true_div" operation, does not copy the stack trace.
So this one probably correctly copies the trace, but it is empty in the first place... I'll investigate more.